source/_integrations/systemmonitor.markdown

```diff
@@ -6,77 +6,58 @@ ha_category:
-file.
-
-| Type (`type:`)         | Argument (`arg:`)         | Argument mandatory        |
-| :--------------------- |:--------------------------|:--------------------------|
-| disk_use_percent       | Path, e.g., `/`           | no                        |
-| disk_use               | Path, e.g., `/`           | no                        |
-| disk_free              | Path, e.g., `/`           | no                        |
-| memory_use_percent     |                           |                           |
-| memory_use             |                           |                           |
-| memory_free            |                           |                           |
-| swap_use_percent       |                           |                           |
-| swap_use               |                           |                           |
-| swap_free              |                           |                           |
-| load_1m                |                           |                           |
-| load_5m                |                           |                           |
-| load_15m               |                           |                           |
-| network_in             | Interface, e.g., `eth0`   | yes                       |
-| network_out            | Interface, e.g., `eth0`   | yes                       |
-| throughput_network_in  | Interface, e.g., `eth0`   | yes                       |
-| throughput_network_out | Interface, e.g., `eth0`   | yes                       |
-| packets_in             | Interface, e.g., `eth0`   | yes                       |
-| packets_out            | Interface, e.g., `eth0`   | yes                       |
-| ipv4_address           | Interface, e.g., `eth0`   | yes                       |
-| ipv6_address           | Interface, e.g., `eth0`   | yes                       |
-| processor_use          |                           |                           |
-| processor_temperature  |                           |                           |
-| process                | Binary, e.g., `octave-cli` | yes                       |
-| last_boot              |                           |                           |
+## Sensors
 
-## Disk usage
+**All entities are disabled by default, you need to enable the entities that you wish to use.**
+
+### Disks
+
+- Disk free - (One per disk/mount point)
+- Disk use - (One per disk/mount point)
+- Disk use (percent) - (One per disk/mount point)
+
+### Network
+
+- IPv4 address - (One per network interface)
+- IPv6 address - (One per network interface)
+- Network in - (One per network interface)
+- Network out - (One per network interface)
+- Packets in - (One per network interface)
+- Packets out - (One per network interface)
+- Network throughput in - (One per network interface)
+- Network throughput out - (One per network interface)
+
+### Other
+
+- Last boot
+- Load (15m)
+- Load (5m)
+- Load (1m)
```

Comment:
I see that other sites use m in this context as well. But would it be wrong to use min instead?

Reply:
The "m" means minute so it wouldn't be wrong. Currently the sensors are named "Load (15m)" etc. in core. We should probably be consistent.

https://github.com/home-assistant/core/blob/fbcb31b1030e51cab2aac95484c4008fab40fbe2/homeassistant/components/systemmonitor/sensor.py#L123

I suggest the renaming to be done separately.